dashboard/app/views/levels/_level_group.haml

```diff
@@ -10,25 +10,29 @@
 = render partial: 'levels/common_audio'
 
 -#
-  If we are looking at the level independent of a script, then let's just
-  show all pages of the LevelGroup at once.  Otherwise we'll show the actual
-  page.  The controller takes care of that for us.
-
-- current_page = params[:puzzle_page] || 1
-
-- @pages.each do |page|
-
-  - unless data["title"].blank?
-    %h2= t('heading_x_of_y', heading: data["title"], x: page.page_number, y: @level.pages.length)
-    %br/
-
-  - page.levels.each_with_index do |level, index|
-
-    -# Show last attempted answer, unless this is a teacher looking at a student's attempt
-    -# at an anonymous survey.  In that case it's very important that we do not show the
-    -# last attempt.
-    -# Check that @script_level exists, because it won't for a /level/ view.
-    - if @script_level && @script_level.can_view_last_attempt(current_user, @user)
+  If the viewer can't view last_attempt, which would happen when it's a teacher
+  looking at a student's attempt at an anonymous survey, then just show explanation
+  text.  It's very important that we don't lose or miss this check.
+  Check that @script_level exists, because it won't for a /level/ view.
```

Comment:
After further looking, this gets tricky because it's actually a view - https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/show.html.haml - that decides which haml to show for a level.  Presumably the bigger fix would be to hoist that logic into controllers so that it can be unit tested.